### DISCUSSION TITLE
Hitting find content before searching for content

### DISCUSSION THREAD

**Discussion by robdekort** [trendyminds/statamic-palette#6] [Ideas]
Something I keep forgetting when searching for content is to go into `Find content` first. I wonder if it all could be one the same root level? If not, I'm sure you have reasons. I'm mostly curious, and sure I can get used to it if this is best.

**Comment by aaronbushnell** [comment]
Hey @robdekort thanks for the feedback! I thought about that approach initially because it would've simplified the front-end a _lot_!

The reason I implemented it this way was because I was worried a site with a sizable set of content would make searching for actions hard. For example, if a client had a few dozen blog posts prefixed with "Collections" or "Field" it might push down the action you're looking for by quite a bit. 

What I _may_ do is make contexts accessible from a keyboard shortcut, too. So—`CMD+K` would be Palette, `CMD+J` (for instance) would be "Find Content".

**Reply by robdekort** [reply]
Hey Aaron, yeah I figured something like that was probably the reason. I like the thought of separate shortcuts. Something else you could consider is having it all in one root but adding the ability to search `content something`, `fieldset something` to narrow stuff down. Anyways, I'll leave it to your good care. Thanks!
